crates/trie/parallel/src/proof.rs

```diff
@@ -10,11 +10,11 @@ use alloy_primitives::{
     B256,
 };
 use alloy_rlp::{BufMut, Encodable};
+use crossbeam_channel::Receiver;
```

Comment:
I wanted to use crossbeam channels for this for a while but never felt necessary because we only have a few hundred or thousand messages here, but with more things to fetch this is definitely beneficial I believe, so we should use crossbeam here